avm/res/network/application-gateway/version.json

```
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+    "version": "0.5",
```

```suggestion
    "version": "0.1",
```

I guess we start low? 😄